### PR TITLE
Feature/baip 1531

### PR DESCRIPTION
**Pull request by mumuchuchu**

### CODE REVIEW

File: src/schema/span-measures-survey/__mocks__/span-measures-survey.repository.ts [hunk 1]
```diff
@@ -2,5 +2,6 @@ import { SpanMeasuresSurvey } from '../__stubs__/span-measures-survey-stub';
 
 export const SpanMeasuresSurveyRepository = jest.fn(() => ({
 	updateSpanMeasuresSurvey: jest.fn(() => SpanMeasuresSurvey),
+	updateSpanMeasuresCompletion: jest.fn(() => SpanMeasuresSurvey),
```
Comment (bennyvdhoogen) [inline src/schema/span-measures-survey/__mocks__/span-measures-survey.repository.ts:5]: The naming on this follows the format, but sounds a bit strange as `completion` kind of rules of `updating`. It depend on how it's named for the other surveys, but what do you think of `completeSpanMeasuresSurvey` as an alternative?

File: src/schema/span-measures-survey/span-measures-survey.repository.spec.ts [hunk 2]
```diff
@@ -37,6 +37,21 @@ describe('Span Measures survey / Repository', () => {
 		);
 	});
 
+	test('updateSpanMeasuresComletion()', async () => {
```
Comment (bennyvdhoogen) [inline src/schema/span-measures-survey/span-measures-survey.repository.spec.ts:40]: typo, should be > updateSpanMeasuresCompletion()

**Comment by bennyvdhoogen** [comment]
Looks good Muchu, just one comment regarding naming + there's a very small typo in one of the tests.

File: src/schema/span-measures-survey/span-measures-survey.repository.spec.ts (hunk 2 above)
Comment (mumuchuchu) [reply src/schema/span-measures-survey/span-measures-survey.repository.spec.ts:40]: good catch!

**Review by bennyvdhoogen — APPROVED** [review]
LGTM!

**Pull request merged**